distributed/protocol/numpy.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import print_function, division, absolute_import
+
+import sys
+
+import numpy as np
+
+try:
+    import blosc
+    n = blosc.set_nthreads(2)
```

Comment:
Yeah, we could place this in the `~/.dask/config` file.  Users can also call `blosc.set_threads` themselves.  It's possible that blosc itself should have some place to make this configurable instead.